src/libplctag/Tag.cs

```diff
@@ -613,10 +636,11 @@ public void Dispose()
                 return;
 
             if (_isInitialized)
-                RemoveEventsAndRemoveCallback();
-
-            var result = (Status)_native.plc_tag_destroy(nativeTagHandle);
-            ThrowIfStatusNotOk(result);
+            {
+                RemoveCallback();
```

Comment:
@kyle-github - this is the method that is used to tear down the tag either explicitly or implicitly by the garbage collector.

Previously, this code was checking that neither removing the event callback, nor destroying the tag, returned an error code, and if so would throw an exception. I'm not sure if there are any valid reasons for return an error code (other than bugs in the wrapper or core library), and I believe there is no recourse if either fail.

The new code just ignores any errors.

Your thoughts?